pom.xml

```diff
@@ -160,7 +160,11 @@
         <exclusions>
           <exclusion>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
+            <artifactId>*</artifactId>
```

Comment:
'*' might be too broad

Reply:
it means that 
1. classic log4j is excluded
2. slf4j-reload4j is excluded
3. when the hadoop move to log4j2 is finally shipped, its slf4j bindings will be excluded too.

exclusions 1 and 2 could be done explicitly, but #3 is unclear until something ships. (hadoop trunk/3.4 is on log4j2, but the move isn't complete)